lib/protocol/init.js

```diff
@@ -56,10 +56,40 @@ export default function init (desiredCapabilities = {}) {
         }
     })
 
-    return this.requestHandler.create({
-        path: '/session',
-        method: 'POST'
-    }, {
-        desiredCapabilities: this.desiredCapabilities
-    })
+    const jsonwpCaps = this.desiredCapabilities;
+    const w3cCapKeys = [
```

Comment:
These don't match what exists in webdriver v5:
https://github.com/webdriverio/webdriverio/blob/e79328564e049fe6c81f0319ffe89d9293462e69/packages/webdriverio/src/constants.js#L369

looking at `goog:chromeOptions` and `moz:firefoxOptions`

Reply:
@emilyrohrbough `goog:chromeOptions` and `moz:firefoxOptions` are valid w3c browser specific options.
See https://www.w3.org/TR/webdriver1/#dfn-extension-capability, https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities.
![image](https://user-images.githubusercontent.com/1262296/63520771-d9347180-c527-11e9-8a28-ccc97564ebf6.png)